examples/react-redux/js/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { render } from 'react-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import { Router, Route, hashHistory } from 'react-router';
+import App from './containers/App';
+import reducer from './reducers';
+import { loadState, saveState } from './storage';
+
+// Loads the initial state from local storage
+const initialState = loadState();
+
+// Creates our store with the './reducers'
+// reducer and the initial state
+const store = createStore(reducer, initialState);
+
+// Saves the state on local storage each time it changes
+store.subscribe(() => saveState(store.getState()));
+
+// Renders the application on the .todoapp element
+render(
+	<Provider store={store}>
+		{/* Provider passes the store to child elements through React context
```

Comment:
I know it's too much for real code, but they will give others a better understanding on what's going on in order to take a good decision about the framework to use.